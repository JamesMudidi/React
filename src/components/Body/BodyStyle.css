.lead {
    margin-right: 25rem !important;
    margin-left: 25rem !important;
    font-size: 2rem !important;
}

.display-4 {
    margin-right: 25rem !important;
    margin-left: 25rem !important;
}

.jumbotron {
    text-align: center !important;
    padding: 15rem 2rem 15rem !important;
    margin-bottom: .1rem !important;
}

.text {
    text-align: center;
    margin-right: 25rem !important;
    margin-left: 25rem !important;
}