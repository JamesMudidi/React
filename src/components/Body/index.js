import React, { Component } from 'react';
import { Jumbotron } from 'react-bootstrap';
import './BodyStyle.css';

class Body extends Component {
  render() {
    return (
      <div>
        <Jumbotron>
          <h1 className="display-4">Hello!</h1>
          <p className="lead">
            Welcome to iRepoter
          </p>
          <hr className="my-2" />
          <p className="text">
            iReporter enables any/every citizen to bring any form of corruption
            to the notice of appropriate authorities and the general public.
            Users can also report on things that need government intervention
            around the community
          </p>
        </Jumbotron>
      </div>
    );
  }
}

export default Body;
