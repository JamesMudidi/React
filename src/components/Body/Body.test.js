import React from 'react';
import { shallow } from 'enzyme';
import Body from 'components/Body';

describe('Body', () => {
  it('matches the snapshot', () => {
    const component = shallow(<Body />);
    expect(component).toMatchSnapshot();
  });
});
