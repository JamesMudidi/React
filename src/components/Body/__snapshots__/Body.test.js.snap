// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body matches the snapshot 1`] = `
<div>
  <Bootstrap(Jumbotron)>
    <h1
      className="display-4"
    >
      Hello!
    </h1>
    <p
      className="lead"
    >
      Welcome to iRepoter
    </p>
    <hr
      className="my-2"
    />
    <p
      className="text"
    >
      iReporter enables any/every citizen to bring any form of corruption to the notice of appropriate authorities and the general public. Users can also report on things that need government intervention around the community
    </p>
  </Bootstrap(Jumbotron)>
</div>
`;
